### DISCUSSION TITLE
uBlock Origin Hack extentions

### DISCUSSION THREAD

**Discussion by jetchg** [3kh0/ext-remover#902] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
On a school Chromebook, how would you be able to use bookmarklets on the extension page, because I can use it to have bookmarklets on most sites, but not on the [webstorex](https://chrome.google.com/webstorex) page, because I want to use it to block GoGuardian and install extensions.

**Comment by S-PScripts** [comment]
you can install extensions?
anyways just use another exploit instead of ltbeef go guardian has plenty of them
